### ISSUE TITLE
Consider adding hidden doors

### ISSUE THREAD

**Issue by R2Dead2** [tgruby/dungeon-hunt#8]
If possible, the hidden doors would not appear on maps.  Purchasing maps would become much more valuable if there might be areas in previous levels that lead to good treasure.

**Comment by tgruby** [comment]
I like it, but hard to do.  Since the maps are random generated, I would have to think about how to do this... also, what happens if you walk through a Hidden Door (presumably a wall)? do you walk through it?

**Comment by R2Dead2** [comment]
Yes.  Think of it like an illusionary wall. Bump into it and instead of getting the message that you can't walk through walls, you are in another room with a chest or (on the more difficult levels) portal rooms.

**Comment by tgruby** [comment]
Got it.
